recipe/meta.yaml

```
@@ -38,21 +38,22 @@ requirements:
     # Correlations
     - phik >=0.11.1,<0.13
     # Examples
-    - requests >=2.24.0,<2.29
+    - requests >=2.24.0,<3
     # Progress bar
-    - tqdm >=4.48.2,<4.65
+    - tqdm >=4.48.2,<5
     - seaborn >=0.10.1,<0.13
-    - multimethod >=1.4,<1.10
+    - multimethod >=1.4,<2
     # metrics
-    - statsmodels >=0.13.2,<0.14
-    - typeguard >=2.13.2,<2.14
+    - statsmodels >=0.13.2,<1
+    # type checking
+    - typeguard >=2.13.2,<3
     - imagehash ==4.3.1
-    - pillow
+    - wordcloud >=1.8.2.2
+    - dacite >=1.6
```

```suggestion
    - dacite >=1.8
```
https://github.com/ydataai/ydata-profiling/blob/116fb8257ff49b0071ac321779739643650fe59e/requirements.txt#L26C1-L26C12

---

Made the amendment! Thanks!